### PR TITLE
Finalized the projects using ES6 modules

### PR DESCRIPTION
**Pull request by muneebulrehman**
Refactored the existing Awesome-books project using the ES6 modules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		"eslint": "^7.32.0",
+		"eslint-config-airbnb-base": "^14.2.1",
+		"eslint-plugin-import": "^2.25.3",
+		"hint": "^6.1.9",
+		"stylelint": "^13.13.1",
+		"stylelint-config-standard": "^21.0.0",
+		"stylelint-csstree-validator": "^1.9.0",
+		"stylelint-scss": "^3.21.0"
+	}
+}
```
Comment (elmar8287) [inline package.json:34]: - [x] Please, consider that I can not find the Luxon package in the list of installed ones in the package.json file. Please, could you check, is it installed correctly, as mentioned in project requirements: `npm install --save luxon`

File: src/nav.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    month: 'long',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+    hour12: true,
+  };
+  const formatedDate = Intl.DateTimeFormat([], obj).format(date);
+  dateEl.textContent = formatedDate;
+}
```
Comment (elmar8287) [inline src/nav.js]: - [x] Please, consider using the arrow function syntax, as required.

File: src/library.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	  this.setLocalStorage();
+	  this.renderList();
+	};
+
+	clickToremove(e) {
+	  if (e.target.classList.contains('remove-btn')) {
+	    const removeItem = +e.target.dataset.id;
+	    this.removeList(removeItem);
+	  }
+	}
```
Comment (elmar8287) [inline src/library.js]: - [x] You can divide your code into additional modules. Here are add and remove methods that could be created separate (in the modules folder you will create) and then used via importing.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @muneebulrehman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  Check the comments under the review.
- [x]  Please, consider to the user the following name for `index` files: `index.html`, `index.js`, and `index.css`, as required in the project. (index.html is ok)
- [x]  Please, consider to create `modeles` directory with other .js files.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/library.js (hunk 3 above)
Comment (muneebulrehman) [reply src/library.js]: I can do so but adding a book and removing a book are core functionality of a library. Moving these methods to standalone functions inside another module would defeat the purpose of object-oriented programming.

File: src/nav.js (hunk 2 above)
Comment (muneebulrehman) [reply src/nav.js]: Done

File: package.json (hunk 1 above)
Comment (muneebulrehman) [reply package.json:34]: Done

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @muneebulrehman,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**